### ISSUE TITLE
Curvilinear Pericallosal Lipoma 

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7096]
**Preferred term label:**

 Curvilinear Pericallosal Lipoma 

**Synonyms**

**Definition (free text, please give PubMed ID)**

Pericallosal lipomas (PCL) are congenital soft masses of adipose cells encapsulated by a thin layer of fibrous tissue, appearing adjacent to the corpus callosum of the brain.  Curvilinear pericallosal lipomas are thin and curve around  the  posterior end  of the splenium.  

https://obgyn.onlinelibrary.wiley.com/doi/pdf/10.1002/uog.17400  as PMID: 28067000
https://n.neurology.org/content/93/2/e212   as PMID: 31285412

Also PMID: 30104143

**Parent term (use hpo.jax.org/app)**
Parent term will be "Pericallosal Lipoma" submitted as a separate issue term request.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pai disease ORPHA:1993 is one syndrome that has these lipomas but not the only manifestation as they can occur on their own without any other symptoms. 

**Your nano-attribution (ORCID)**
0000-0002-3302-4610

**Comment by pnrobinson** [comment]
added term as HP_0034013
